forgerock-auth/src/main/java/org/forgerock/android/auth/FRUser.java

```diff
@@ -298,17 +297,6 @@ public void login(Fragment fragment, FRListener<FRUser> listener) {
          *                 <b> throws {@link java.net.MalformedURLException} When failed to parse the URL for API request.
          */
         public void login(FragmentActivity activity, FRListener<FRUser> listener) {
```

Comment:
For compose project , i think people mix and match ComponentActivity or AppCompatActivity

(AppCompatActivity extends FragmentActivity extends ComponentActivity)

So , Do you think we can expose a new method -  public void login(ComponentActivity activity, FRListener<FRUser> listener)

Reply:
The function registerForActivityResult() requires to run inside onCreate, after creation, when calling registerForActivityResult will throw an exception, the method registerBrowserLauncher -> registerForActivityResult, so it has to run inside onCreate(). I tried not to change the existing interface, and typically App Activity should either extend AppCompatActivity or FragmentActivity.